components/gitpod-protocol/go/reconnecting-ws.go

```diff
@@ -124,6 +124,11 @@ func (rc *ReconnectingWebsocket) ReadObject(v interface{}) error {
 
 // Dial creates a new client connection.
 func (rc *ReconnectingWebsocket) Dial() {
```

Comment:
I don't think we have clients using it, can we remove?

Reply:
I was mirroring the underlying websocket which uses the same approach. In hindsight likely for b/w compat 🤷  which we don't really need to support.

Will do.